.github/workflows/pull-translations.yml

```diff
@@ -0,0 +1,74 @@
+name: Pull & Commit Translations from Mojito
+
+on:
+  schedule:
+    # Run every day at 9am and 9pm ET.
+    # Note: We only run every 12 hours, otherwise our Mojito webapp (on Heroku)
+    # wouldn't be able to sleep, and would go over its free hours.
+    - cron: '0 13,1 * * *'
+
+jobs:
+  pull-translations:
+    if: github.repository_owner == 'patcon'
+    runs-on: ubuntu-latest
+    env:
+      BRANCH: 320-continuous-localization
+      CHECKOUT_DIR: code
+      MOJITO_VERSION: "0.110"
+      MOJITO_REPO: polis
```

Comment:
I need to update this to `polis-client-participation`, so we can more easily add localization for other components.

I should also write docs for administrating this, e.g. setting up a new repo.